modules/apps/frontend-data-set/frontend-data-set-test/src/testFunctional/tests/FDSFilters.testcase

```diff
@@ -107,4 +107,40 @@ definition {
 		}
 	}
 
+	@description = "LPS-155308. Assert one filter can be removed."
+	@priority = "5"
+	test OneFilterCanBeRemoved {
+		task ("When there is a filter already established") {
+			AssertElementPresent(
+				key_filter = "Status: Approved, Draft",
+				locator1 = "FrontendDataSet#FILTER_SUMMARY_LABEL");
+		}
+
+		task ("Then it can be removed one by one") {
+			ClickNoError(
+				key_filter = "Status: Approved, Draft",
+				locator1 = "FrontendDataSet#FILTER_SUMMARY_LABEL");
+
+			FDSFilters.disableStatusFilters(key_status = "Approved");
+
+			Click(locator1 = "FrontendDataSet#EDIT_FILTER_BUTTON");
+
+			AssertElementPresent(locator1 = "FrontendDataSet#EMPTY_FDS_TABLE_MESSAGE");
+
+			FDSFilters.disableStatusFilters(key_status = "Draft");
+
+			Click(locator1 = "FrontendDataSet#DELETE_FILTER_BUTTON");
+		}
+
+		task ("And the results will be displayed updated accordingly to it") {
+			AssertElementPresent(locator1 = "FrontendDataSet#TABLE_BODY");
+		}
```

Comment:
I think it's better to check if all 100 results show up after removing the filters. Maybe we can assert `Showing 1 to 10 of 100` is present.